### PR TITLE
CITY-13: Add a post about the schools co-design activity

### PR DESCRIPTION
**Pull request by sepidehshahi**
New post about the schools co-design activity.

### CODE REVIEW

**Comment by sepidehshahi** [comment]
@simonbates and @colinbdclark please review this PR. Thanks

File: index.html [hunk 1]
```diff
@@ -91,10 +91,17 @@ <h1>Co-designing Inclusive Cities</h1><br />
...
+          <div class="ideasLink"><h3><a href="ideas/students-co-design-activity">Students co-designing their dream neighbourhoods</a></h3></div>
+          <div class="ideasDesc"><p>The cities being built today are the cities our children will be living in as adults. However, children are seldom included in urban design and planning conversations and processes. <a href="ideas/students-co-design-activity">Read more</a>.</p></div>
+      </div>
+      <div class="ideasBlock">
 				<div class="ideasDate">May 10, 2018</div>
 				<div class="ideasLink"><h3><a href="ideas/co-design-draft-proposal">IDRC Inclusive Co-Design Draft Proposal</a></h3></div>
-				<div class="ideasDesc"><p>Last week at their public roundtable meeting, Sidewalk Toronto announced a new partnership with OCAD University to address accessibility issues in their Quayside project. <a href="ideas/for-an-inclusive-quayside">Read more</a>.</p></div>
+				<div class="ideasDesc"><p>Last week at their public roundtable meeting, Sidewalk Toronto announced a new partnership with OCAD University to address accessibility issues in their Quayside project. <a href="ideas/co-design-draft-proposal">Read more</a>.</p></div>
```
Comment (simonbates) [inline index.html]: Good catch!

File: index.html [hunk 2]
```diff
@@ -91,10 +91,17 @@ <h1>Co-designing Inclusive Cities</h1><br />
 	<div id="ideasContainer">
 		<div class="ideas">
 			<h2>Ideas in Progress</h2>
-			<div class="ideasBlock">
+      <div class="ideas">
+        <h2>Ideas in Progress</h2>
```
Comment (simonbates) [inline index.html]: "Ideas in Progress" is duplicated:

```
<div class="ideas">
    <h2>Ideas in Progress</h2>
```

File: index.html [hunk 3]
```diff
@@ -91,10 +91,17 @@ <h1>Co-designing Inclusive Cities</h1><br />
...
+        <div class="ideasBlock">
```
Comment (simonbates) [inline index.html]: This file is (mostly) indented with tabs. Please intent your changes with tabs also, to make consistent.

I'm thinking we should switch the whole file to spaces at some point, but for now please use tabs to be consistent.

**Comment by simonbates** [comment]
We have permission to use `images/StudentsActivityBoardInProgress.png`?

File: ideas/students-co-design-activity/index.html [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+                  <p>During the activities, students are divided into small groups and given an activity board. The board includes a blank map of the neighbourhood and a list of 20 feature icons. Each group gets to discuss the features and vote for the top ten to include in their neighbourhood. Once the features are selected, groups can exchange their activity boards to consider pros and cons of the features selected by their peers. Once done, groups can move back to their original activity board, discuss the pros and cons and decide whether they want to change any of their features based on their peers’ feedback. As a final activity, students are given a development proposal that encourages them to think more critically about other important urban issues, such as the ethics of gathering, owning and using digital data.</p>
```
Comment (simonbates) [inline ideas/students-co-design-activity/index.html]: Some minor text suggestions:

- Add ", and added to their maps" to "Once the features are selected, groups can exchange their activity boards to consider pros and cons of the features selected by their peers.":
    - Once the features are selected, and added to their maps, groups can exchange their activity boards to consider pros and cons of the features selected by their peers.
- Add a comma after "owning" in the final sentence:
    - ...such as the ethics of gathering, owning, and using digital data.

File: ideas/students-co-design-activity/index.html [hunk 5]
```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Students co-designing their dream neighbourhoods</title>
```
Comment (simonbates) [inline ideas/students-co-design-activity/index.html]: Please add "Co-designing Inclusive Cities - " to the title

File: ideas/students-co-design-activity/index.html [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+                  <p>During the activities, students are divided into small groups and given an activity board. The board includes a blank map of the neighbourhood and a list of 20 feature icons. Each group gets to discuss the features and vote for the top ten to include in their neighbourhood. Once the features are selected, groups can exchange their activity boards to consider pros and cons of the features selected by their peers. Once done, groups can move back to their original activity board, discuss the pros and cons and decide whether they want to change any of their features based on their peers’ feedback. As a final activity, students are given a development proposal that encourages them to think more critically about other important urban issues, such as the ethics of gathering, owning and using digital data.</p>
+
+                  <div id="Image">
+
+                      <div class="ImageContainer">
+                          <div class="ImageContainer">
+                          <h4>Students activity board</h2></div>
```
Comment (simonbates) [inline ideas/students-co-design-activity/index.html]: Opens as `h4` but closes as `h2`

File: ideas/students-co-design-activity/index.html [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+                          <div class="imageContainer"><img src="../../images/StudentsActivityBoard.jpg" class="img" alt="StudentsActivityBoard" /></div>
+                      </div>
+                  </div>
+                  <div id="Image">
+
+                      <div class="ImageContainer">
+                          <div class="ImageContainer">
+                          <h4>Students activity board in progress</h2></div>
```
Comment (simonbates) [inline ideas/students-co-design-activity/index.html]: Opens as `h4` but closes as `h2`

File: ideas/students-co-design-activity/index.html [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+                  <p>During the activities, students are divided into small groups and given an activity board. The board includes a blank map of the neighbourhood and a list of 20 feature icons. Each group gets to discuss the features and vote for the top ten to include in their neighbourhood. Once the features are selected, groups can exchange their activity boards to consider pros and cons of the features selected by their peers. Once done, groups can move back to their original activity board, discuss the pros and cons and decide whether they want to change any of their features based on their peers’ feedback. As a final activity, students are given a development proposal that encourages them to think more critically about other important urban issues, such as the ethics of gathering, owning and using digital data.</p>
+
+                  <div id="Image">
+
+                      <div class="ImageContainer">
```
Comment (simonbates) [inline ideas/students-co-design-activity/index.html]: I think we don't have a class `"ImageContainer"` (uppercase I) only lowercase "i" imageContainer. So I think we can remove the `<div class="ImageContainer">` markup.

**Comment by simonbates** [comment]
I've created a JIRA issue for this work:

https://issues.fluidproject.org/browse/CITY-13

Please prefix commit comments for future commits for this pull request with "[CITY-13]: ".

File: ideas/students-co-design-activity/index.html [hunk 9]
```diff
@@ -0,0 +1,115 @@
...
+                  <p>During the activities, students are divided into small groups and given an activity board. The board includes a blank map of the neighbourhood and a list of 20 feature icons. Each group gets to discuss the features and vote for the top ten to include in their neighbourhood. Once the features are selected, groups can exchange their activity boards to consider pros and cons of the features selected by their peers. Once done, groups can move back to their original activity board, discuss the pros and cons and decide whether they want to change any of their features based on their peers’ feedback. As a final activity, students are given a development proposal that encourages them to think more critically about other important urban issues, such as the ethics of gathering, owning and using digital data.</p>
+
+                  <div id="Image">
```
Comment (simonbates) [inline ideas/students-co-design-activity/index.html]: Let's remove the `<div id="Image">` divs also.

File: index.html [hunk 10]
```diff
@@ -91,11 +91,19 @@ <h1>Co-designing Inclusive Cities</h1><br />
 	<div id="ideasContainer">
 		<div class="ideas">
 			<h2>Ideas in Progress</h2>
+		<div class="ideas">
```
Comment (simonbates) [inline index.html]: This line is duplicated and can be removed

**Pull request merged**